### PR TITLE
[android/build] Build tensor_query with nnstreamer-edge sources @open sesame 11/15 11:17

### PR DESCRIPTION
**Pull request by anyj0527**
- Define source files to build tensor_query elements.
- User should set envvar "NNSTREAMER_EDGE_ROOT" (path to nnstreamer-edge source dir) to build tensor_query.
- Add a simple Pipeline API TC for tensor_query.

This PR requires https://github.com/nnstreamer/nnstreamer-edge/pull/113 and https://github.com/nnstreamer/TAOS-CI/pull/739 to be merged

Signed-off-by: Yongjoo Ahn <yongjoo1.ahn@samsung.com>

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by myungjoo — APPROVED** [review]
LGTM

File: java/android/nnstreamer/src/androidTest/java/org/nnsuite/nnstreamer/APITestPipeline.java [hunk 1]
```diff
@@ -2386,4 +2387,64 @@ public void onNewDataReceived(TensorsData data) {
             fail();
         }
     }
+
+    private static int getAvailablePort() {
```
Comment (jaeyun-jung) [inline java/android/nnstreamer/src/androidTest/java/org/nnsuite/nnstreamer/APITestPipeline.java:2391]: let's move this to common util

Comment (anyj0527) [reply java/android/nnstreamer/src/androidTest/java/org/nnsuite/nnstreamer/APITestPipeline.java]: Updated, Thanks!

File: java/build-nnstreamer-android.sh [hunk 2]
```diff
@@ -467,8 +492,9 @@ if [[ $include_assets == "yes" ]]; then
     sed -i "s|GSTREAMER_INCLUDE_CA_CERTIFICATES := no|GSTREAMER_INCLUDE_CA_CERTIFICATES := yes|" nnstreamer/src/main/jni/Android.mk
 fi
 
-# Update ML_API, NNStreamer, GStreamer and Android SDK
+# Update ML_API, nnstreamer-edge, NNStreamer, GStreamer and Android SDK
 sed -i "s|mlApiRoot=ml-api-path|mlApiRoot=$ml_api_dir|" gradle.properties
+sed -i "s|nnstreamerEdgeRoot=nnstreamer-edge-path|nnstreamerEdgeRoot=$nnstreamer_edge_dir|" gradle.properties
```
Comment (jaeyun-jung) [inline java/build-nnstreamer-android.sh:497]: please update gradle.properties in nnstreamer-android-res repo later.

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**